scenarioo-client/test/spec/shared/navigation/branchOrderBy.filter.spec.js

```diff
@@ -170,8 +233,56 @@ describe('Filter scBranchOrderBy', function () {
             expect(result[2].branch.name).toEqual('Ba');
             expect(result[3].branch.name).toEqual('Ae');
         });
+    });
+
+    describe('should order given branch resource objects by name-ascending:', function () {
+
+        it('initConfig', function () {
```

Comment:
why isnt that a beforeEach ? that is strange to me, cause that is not really a test on its own but just the "given"-clause kind of. also in other places.